### PR TITLE
[zh-TW] update the translate of CustomEvent

### PR DESCRIPTION
**Pull request by shiyou0130011**
I changed `Web/API/CustomEvent` and `Web/API/CustomEvent/CustomEvent` to **Markdown format** and updated them to follow the newest English doc.

### CODE REVIEW

**Review by yin1999 — CHANGES_REQUESTED** [review]
Please leave 2 commits for a markdown convertion (reference: <https://github.com/mdn/translated-content/pull/6153/commits>). You can do the `update` task in the second commit (convert and sync)

**Comment by shiyou0130011** [comment]
> Please leave 2 commits for a markdown convertion (reference: https://github.com/mdn/translated-content/pull/6153/commits). You can do the `update` task in the second commit (convert and sync)

Hello [yin1999](https://github.com/yin1999)
I've update the branch to leave only 2 commits. Please check.

File: files/zh-tw/web/api/customevent/customevent/index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    return evt;
+   }
+
+  CustomEvent.prototype = window.Event.prototype;
+
+  window.CustomEvent = CustomEvent;
+})();
+```
+
+## 延伸閱讀
```
Comment (yin1999) [inline files/zh-tw/web/api/customevent/customevent/index.md:85]:
```suggestion
## 參見
```

File: files/zh-tw/web/api/customevent/customevent/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+-  {{domxref("CustomEvent")}}
```
Comment (yin1999) [inline files/zh-tw/web/api/customevent/customevent/index.md:87]:
```suggestion
- {{domxref("CustomEvent")}}
```

File: files/zh-tw/web/api/customevent/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## 屬性
+
+_此介面繼承了其父介面 {{domxref("Event")}} 的屬性_
+
+
+- {{domxref("CustomEvent.initCustomEvent()")}} {{deprecated_inline}}
+  - : 初始化一個`CustomEvent`物件。若該事件已經被觸發，則不會進行任何動作。
+
+
```
Comment (yin1999) [inline files/zh-tw/web/api/customevent/index.md:30]:
```suggestion

```

File: files/zh-tw/web/api/customevent/index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+## 規格
+
+{{Specifications}}
+
+## 瀏覽器兼容性
+
+
```
Comment (yin1999) [inline files/zh-tw/web/api/customevent/index.md:37]:
```suggestion

```

File: files/zh-tw/web/api/customevent/index.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+{{Compat}}
+
+
+## 延伸閱讀
```
Comment (yin1999) [inline files/zh-tw/web/api/customevent/index.md:41]:
```suggestion

## 參見
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
I am not familiar with traditional Chinese, so just for some typo (still in 2 commits).

**Review by irvin — APPROVED** [review]

**Comment by yin1999** [comment]
> @shiyou0130011 conflict 了，需要 rebase

resolved

**Pull request merged**